src/hooks/video/agora/index.ts

```diff
@@ -0,0 +1,233 @@
+import { useState, useEffect, useCallback } from "react";
+import AgoraRTC, {
+  IAgoraRTCRemoteUser,
+  ILocalAudioTrack,
+  ILocalVideoTrack,
+} from "agora-rtc-sdk-ng";
+
+import { AGORA_APP_ID, AGORA_CHANNEL, AGORA_TOKEN } from "secrets";
```

Comment:
Calling out usages of `AGORA_APP_ID`, `AGORA_CHANNEL`, and `AGORA_TOKEN` that are likely wrong and almost certainly will need refactoring.